app/api/dao/user.py

```diff
@@ -170,14 +173,19 @@ def list_users(
 
         """
 
-        users_list = (
-            UserModel.query.filter(
-                UserModel.id != user_id,
-                not is_verified or UserModel.is_email_verified,
-                func.lower(UserModel.name).contains(search_query.lower())
-                | func.lower(UserModel.username).contains(search_query.lower()),
+        users_list_query = UserModel.query.filter(
+            UserModel.id != user_id,
+            not is_verified or UserModel.is_email_verified,
+            func.lower(UserModel.name).contains(search_query.lower())
+            | func.lower(UserModel.username).contains(search_query.lower()),
+        )
+        if skill:
+            ps = PorterStemmer()
+            users_list_query = users_list_query.filter(
+                func.lower(UserModel.skills).contains(ps.stem(skill.lower()))
```

Comment:
This is not where is should be done. It's like this for now I guess. Please revert the change.

Reply:
Hi @epicadk. I didn't quite understand your comment.

I came across a [blog](https://niallburkley.com/blog/index-columns-for-like-in-postgres/) which says that ilike will use indexes if created. So, I tried achieving the functionality we need by using a stemmer. I'm still exploring the FTS part.

Reply:
Yes that's find I'll open another issue for fts. Please revert the porter changes for now. The reset looks good.